src/VisualStudio/Core/Def/Commands.vsct

```diff
@@ -411,6 +424,7 @@
       <IDSymbol name="IDG_CSHARP_CODEGEN" value="0x3490"/>
 
       <IDSymbol name="cmdidContextOrganizeRemoveAndSort" value="0x1913"/>
+      <IDSymbol name="cmdidCSharpOrganizeSortUsings" value="0x1922"/>
```

Comment:
Curious: is this reusing the value of the old command we previous removed? If so, awesome, since people who wrote extensions using that will be happy.

Reply:
Indeed. It is the same id.